src/server/create/helpers/audit-created.js

```diff
@@ -0,0 +1,7 @@
+async function auditCreated(request, repositoryKind, repositoryName) {
+  request.audit.send(
+    `${repositoryKind} created: ${repositoryName} by ${request.pre?.authedUser.id}:${request.pre?.authedUser.email}`
```

Comment:
A few things:
- I don't think we need `provideAuthedUser` with this? We should only send a message with authed user information if the user is logged in. 
- You can obtain user info via `request.getUserSession()` have a look around the codebase theres loads of prior art around this
- You can wrap the audit send in a 
```
const authedUser = await request.getUserSession()
authedUser.isAuthenticated
```

Reply:
My first solution was using `request.getUserSession()` or `request.auth.credentials`. But that was much more code all over the place when `provideAuthedUser` is a much cleaner solution?

though I think I will change the method signature to be less tied to the request.